src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -158,15 +168,18 @@
                 return outOfDateResources
             },
             outOfDateMessage() {
+                const updatedNames = Object.keys(this.outOfDateResources)
+                    .map(r => names[r as keyof DatasetResourceSet]).join(", ")
                 if (Object.keys(this.outOfDateResources).length == 0) {
                     return ""
                 }
-                return "This dataset has been updated in the ADR. Use the refresh button to import the latest files."
+                return `The following files have been updated in the ADR: ${updatedNames}. Use the refresh button to import the latest files.`
```

Comment:
This ought to come from translations file I guess. Not sure how we usually deal with string templates in the translations, have we avoided that issue up til now?

Reply:
It's straight-forward with `i18next`, see https://github.com/mrc-ide/hint/pull/336/files#diff-305dda242c365630c350e025be821a5cR113 for example.